.devcontainer/Dockerfile

```diff
@@ -1,23 +1,9 @@
-# Use a base image that includes both Go and Node.js, such as the official Go image
-FROM golang:1.21.7-bullseye
+ARG AVALANCHEGO_VERSION
+FROM avaplatform/avalanche-cli:v1.4.2 as avalanche-cli
```

Comment:
can we move this version to devcontainer as well?

Reply:
Avalanche CLI now also offers a latest tag, so no more version here